### ISSUE TITLE
Chardet encoding errors crashing screen

### ISSUE THREAD

**Issue by omarkamalakis** [ranger/ranger#1561]
#### Runtime Environment

- Operating system and version: 
- Terminal emulator and version: 
- Python version: 
- Ranger version/commit: 
- Locale: 

#### Steps to reproduce
1. 
2. 
3. 

#### Traceback

```

```

**Comment by toonn** [comment]
Hmm, could you fill out all the required information? Impossible to debug like this.

**Comment by omarkamalakis** [comment]
My bad, I hit return too soon

**Comment by toonn** [comment]
Could you try reproducing this on the master version of ranger? I can't reproduce it. Are the files this happens on in a specific language?

**Comment by omarkamalakis** [comment]
Will do. The errors don't seem to be consistent across languages. Rmarkdown files will often crash, as well as python, javascript, and bash files, but they will also render just fine at other times. It does crash more often when looking at files with non-latin characters.

**Comment by pvl1** [comment]
can you try to back up ur config dir, and run without a config or make a new/fresh config?

**Comment by toonn** [comment]
You can achieve the same effect by running `ranger --clean`, it'll keep ranger from loading your configuration. You might have to unset `RANGER_LOAD_DEFAULT_RC` if you set that to `FALSE`.

**Comment by omarkamalakis** [comment]
Running `ranger --clean` returns the same errors. I am suspicious that it is something unique to my machine. Is there a way I can turn off ranger notifications entirely?

**Comment by toonn** [comment]
No, normally these wouldn't be shown to the user in the first place. So they're already escaping ranger's grasp.

**Comment by lyjdwh** [comment]
I fix it through #1138

**Comment by toonn** [comment]
@lyjdwh, so something you installed changed the debug level? Why does python allow this : /

Way forward seems like we should do a better job of capturing error output, not sure where to start on that though.

**Comment by lyjdwh** [comment]
@toonn 
I installed vext through pip which cased this error in my case.
Maybe you can research what vext does.
